apps/cookbook/src/app/examples/radio-example/examples/template-driven-forms.ts

```diff
@@ -45,46 +39,26 @@ const config = {
       font-style: italic;
     }`,
   ],
-  codeSnippet: `
-items = [
-  { title: 'Bacon', value: 1 },
-  { title: 'Salami', value: 2 },
-  { title: 'Tenderloin', value: 3 },
-  { title: 'Tongue', value: 4 },
-  { title: 'Drumstick', value: 5 },
+  codeSnippet: `items = [
```

Comment:
Consider doing this instead: https://github.com/kirbydesign/designsystem/blob/master/apps/cookbook/src/app/examples/radio-example/examples/custom.ts#L30

Reply:
Clever, Thanks! Implemented.